solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -230,161 +258,153 @@ def local_train(train_file_pattern,
         will create three DNN layers where the first layer will have 10 nodes, 
         the middle layer will have 3 nodes, and the laster layer will have 2
         nodes.
+     learning_rate: tf.train.AdamOptimizer's learning rate,
+     epsilon: tf.train.AdamOptimizer's epsilon value.
   """
-  #TODO(brandondutra): allow other flags to be set like batch size/learner rate
+  if len(train_dataset.input_files) != 1 or len(eval_dataset.input_files) != 1:
+    raise ValueError('CsvDataSets must be built with a file pattern, not list '
+                     'of files.')
+
+  if file_io.file_exists(output_dir):
+    raise ValueError('output_dir already exist. Use a new output path.')
```

Comment:
Kinda, Lets say training is ran with max_steps=1000. User would have to run again with max_steps=1100 to perform an additional 100 steps (not an additional 1100 steps). This is confusing, so I don't allow it. Once I move away from tf.learn I can do this.